dotnet/src/SemanticKernel.Abstractions/AI/PromptExecutionSettings.cs

```diff
@@ -93,7 +135,8 @@ public virtual PromptExecutionSettings Clone()
         return new()
         {
             ModelId = this.ModelId,
-            ExtensionData = this.ExtensionData is not null ? new Dictionary<string, object>(this.ExtensionData) : null
+            ExtensionData = this.ExtensionData is not null ? new Dictionary<string, object>(this.ExtensionData) : null,
+            FunctionChoiceBehavior = this.FunctionChoiceBehavior
```

Comment:
nit. Properties first.

```suggestion
            FunctionChoiceBehavior = this.FunctionChoiceBehavior,
            ExtensionData = this.ExtensionData is not null ? new Dictionary<string, object>(this.ExtensionData) : null,
```

Reply:
fixed